src/compas_fab/ghpython/components/assembly_info.py

```diff
@@ -0,0 +1,22 @@
+import GhPython
+import System
+
+from compas_fab.ghpython.components import assembly_name
+from compas_fab.ghpython.components import assembly_version
+
+
+class AssemblyInfo(GhPython.Assemblies.PythonAssemblyInfo):
+    def get_AssemblyName(self):
+        return assembly_name
+
+    def get_AssemblyDescription(self):
+        return """Components for the COMPAS FAB package."""
+
+    def get_AssemblyVersion(self):
+        return assembly_version
+
+    def get_AuthorName(self):
+        return "rust@arch.ethz.ch"
```

Comment:
Here you could use the generic `gramaziokohler@arch.ethz.ch` email, but also, you could do `from compas_fab.__version__ import __author_email__` and use that instead.